github-actions/verify-pr-title/dist/index.js

```diff
@@ -162,28 +162,62 @@ exports.default = _default;
 "use strict";
 
 Object.defineProperty(exports, "__esModule", { value: true });
-exports.verifyPRTitleForBadTitle = exports.verifyPRTitleForBugId = void 0;
-// Matches @W-0000@ through @W-99999999@
-const reBugId = /@W-\d{4,8}@/;
-// Matches r/ R/ d/ D/
-const reBranch = /^[rdRD]\//;
+exports.verifyReleaseBranchPrTitle = exports.verifyDevBranchPrTitle = void 0;
 /**
- * Verifies title conforms to the pattern required by git2gus.
- * git2gus requires the WorkItemId to be enclosed by two '@' signs.
- * A valid WorkItemId consists of 'W-' followed by 4-8 digits.
+ * This regex portion matches @W-0000@ through @W-999999999@,
+ * to enforce that the title contains a work record number consumable
+ * by Git2Gus.
+ * All pull request titles will require this segment.
  */
-function verifyPRTitleForBugId(title) {
-    return reBugId.test(title);
+const WORK_ITEM_SEGMENT = "@W-\\d{4,8}@";
+/**
+ * This is a regex segment that matches the accepted options for feature branch PR type.
+ * NOTE: This segment allows for exactly one type.
+ */
+const FEATURE_TYPE_SEGMENT = "(NEW|FIX|CHANGE)";
+/**
+ * These are all the possible options for feature branch PR Scope.
+ */
+const FEATURE_SCOPE_OPTIONS = [
+    "CodeAnalyzer",
```

Comment:
Can we have a scope to use when none of these fit? Like "Other"?

Reply:
@rmohan20 So are you thinking that we need "Other" because we might add more engines in future and then don't need to add additional feature scope options? I'm fine with adding Other but wanted to know an example use case.
cc: @jfeingold35